ivy_lint/formatters/docs_formatter.py

```diff
@@ -0,0 +1,70 @@
+import re
+import ast
+
+from ivy_lint.formatters import BaseFormatter
+
+
+def format_docstring(doc):
+    """Formats a single docstring."""
+    # Rename "Functional Examples" to "Examples" and format it without the extra newline
+    doc = re.sub(r'(\s*)Functional Examples\n\1-*\n?', r'\1Examples\n\1--------\n', doc)
+
+    # Ensure newline and correct indentation after "Examples" when it's already there
+    doc = re.sub(r'(\s*)Examples\n\1--------\s*([^\n])', r'\1Examples\n\1--------\n\1\2', doc)
```

Comment:
@lucasalavapena, it was not done intentionally. It happened due to the very first code block in the `Examples` section since we were adding new line characters above each code block. I've now resolved this issue.